python-checks/src/test/resources/checks/cdk/iamPrivilegeEscalation/fromJson.py

```diff
@@ -0,0 +1,62 @@
+from aws_cdk import aws_iam as iam
+
+iam.PolicyStatement.from_json({
+        'Action': 'iam:UpdateLoginProfile',
+    #   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^> {{Permissions are granted on all resources.}}
+        'Effect': 'Allow',
+# Noncompliant@+1 {{This policy is vulnerable to the "Update Login Profile " privilege escalation vector. Remove permissions or restrict the set of resources they apply to.}}
+        'Resource': '*'
+    #   ^^^^^^^^^^^^^^^
+    })
+
+iam.PolicyStatement.from_json({
+        'Action': 'iam:UpdateLoginProfile',
+        'Effect': 'Allow',
+        'Resource': ['*']  # Noncompliant
```

Comment:
Why there is no message after `# Noncompliant`?

Reply:
I'm usually not repeating the message if it's identical to the previous one. WDYT?

Reply:
ahhh cool I didn't know it can be omitted in that case